src/plugins/unified_histogram/public/chart/breakdown_field_selector.tsx

```diff
@@ -79,6 +82,7 @@ export const BreakdownFieldSelector = ({
         options={fieldOptions}
         selectedOptions={selectedFields}
         onChange={onFieldChange}
+        truncationProps={{ truncation: 'middle' }}
```

Comment:
We could also define it as a const since it does not change.

Reply:
Done! I did the same for the singleSelection prop 👌🏼